### DISCUSSION TITLE
“MaximumTXPower”

### DISCUSSION THREAD

**Discussion by inventor7777** [EdgeTX/edgetx#5093] [Q&A] [answered]
Hi all (for literally the tenth time this week (sorry for asking 1 million questions 😂))

I see that there is a sound file on my MT12 that says “Maximum TX Power”
So can the TX power be adjusted manually, or is it all automatic? Or is it just color screen radios? Or is that one of the functions of the Multi module LUA configuration?

thanks!

**Comment by pfeerick** [comment]
Configurable transmit power is a function the RF system... i.e. Crossfire, ELRS, R9 all have adjustable power levels... but you are probably using ELRS, so want to look at https://www.expresslrs.org/software/dynamic-transmit-power/ for more detail ;)

**Reply by inventor7777** [reply]
Thanks! However, I’m using the Multi-module; so I should probably look at the Multi-module guide. Thanks for your help, as I thought it was built into EdgeTX.

**Comment by pfeerick** [comment]
MPM does have a "low power" option, but not sure how much of a power
reduction it is.

**Reply by inventor7777** [reply]
Interesting! Where can I find more details about it? Thanks for info.

**Reply by inventor7777** [reply]
Nvm, I remembered where to do that. It’s been a while since I’ve added a model so I forgot. Thanks for the help!

**Comment by pfeerick** [comment, marked as answer]
Looks like it's 1/10th output power... i.e good for indoor/test bench/short
range stuff.

https://www.multi-module.org/using-the-module/protocol-options#low-power-mode

Don't forget the EdgeTX manual if you're searching head trying to figure
out how to do stuff ;)

https://manual.edgetx.org
